docs/compose/FloatingActionButton/README.md

```diff
@@ -0,0 +1,21 @@
+# Floating Action Button
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+Example of a `BpkFab`:
+
+```Kotlin
+import net.skyscanner.backpack.compose.button.BpkFab
+
+BpkFab(
```

Comment:
So the way we arrange the samples is such that after copy-pasting it transforms into a valid code.

This sample will not do it – because we didn't design the BpkFab to work in a such way. It doesn't have any content lamba, unlike the Fab from Material Design. You code already features some usage of BpkFab (in stories or tests), so you can use it as a reference for the sample.

Reply:
This code will not work if you copy-paste it into any Composable code – we still need to make it working :)